index.html

```diff
@@ -64,12 +97,41 @@ <h3 class="program__card__title">Lecture</h3>
                 <p class="program__card__description">The ponent it's going to talk about industry top level topic.</p>
             </div>
         </div>
+        <a class="program__desktop-link display-none" href="#">SEE THE WHOLE PROGRAM</a>
        <div class="program__join-button">
             <p>Join Web Development Conference</p>
        </div>
     </main>
+    <section class="speakers">
+        <h2 class="subtitle-black">Featured Speakers</h2>
+        <hr class="subtitle__line">
+        <div class="speakers__cards-container" id="speakers-container">
+        </div>
+        <div class="speakers__more">
+            <p>MORE</p>
+            <span><i class="fa-solid fa-angle-down fa-2x"></i></span>
+        </div>
+    </section>
     <footer class="footer">
-
+        <h2 class="subtitle">Partner</h2>
+        <hr class="subtitle__line">
+        <div class="footer__logo-container">
+            <img class="footer__logo mozilla" src="./img/mozilla.png" alt="Google logo">
+            <img class="footer__logo"  src="./img/airbnb.png" alt="Airbnb logo">
+            <img class="footer__logo"  src="./img/google-logo.svg" alt="">
+            <img class="footer__logo"  src="./img/microsoft-logo.svg" alt="">
+            <img class="footer__logo"  src="./img/apple.png" alt="">
+        </div>
     </footer>
+    <div class="license">
```

Comment:
-Kindly use `<footer>` for wrapping this element instead of div. The section above can be wrapped with `<section>` element.